.github/workflows/scripts/benchmark/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM node:18
+
+# Define build-time arguments
+ARG input_cpus
+ARG input_memory
+ARG input_indexer_version
+
+ENV INDEXER_VERSION=${input_indexer_version}
+ENV CPUS=${input_cpus}
+ENV MEMORY=${input_memory}
+
+ENV DB_USER=postgres
+ENV DB_PASS=postgres
+ENV DB_DATABASE=postgres
+ENV DB_HOST=postgres
+ENV DB_PORT=5432
+
+
+# Set CPU shares and memory limit based on build arguments
+CMD ["--cpus=$CPUS", "--memory=$MEMORY"]
```

Comment:
Does this work? Are we sure it doesn't just set the env var which would now report differently to the number of cpu?

My understanding is that these are set like `docker run --cpus=2 --memory=4g <rest-of-command>` which would mean setting here https://github.com/subquery/subql/blob/063701c727b733fd0480b6aa3671551863f0c73b/.github/workflows/benchmark.yml#L107

Reply:
it been part of build options https://docs.docker.com/engine/reference/commandline/build/#options, and when `docker run` it would use this as part of the args. 

I can change this so easy for people to read.